### DISCUSSION TITLE
Is there a `StrictVersion` after we drop `distutils`?

### DISCUSSION THREAD

**Discussion by cclauss** [pypa/setuptools#3145] [Q&A]
We currently use `from distutils.version import StrictVersion` but we want to drop `distutils` in preparation for Python 3.12.  What is the recommended substitute for `StrictVersion`?  https://github.com/python/cpython/blob/main/Lib/distutils/version.py#L93

https://github.com/pypa/packaging/blob/main/packaging/version.py

**Comment by Wouterkoorn** [comment]
Maybe worth raising an issue over at https://github.com/pypa/packaging/issues

**Comment by jiridanek** [comment]
* https://github.com/pypa/packaging/issues/520
* https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python

I went with

```python
from pkg_resources import parse_version
```

**Reply by l0b0** [reply]
It looks like the current recommendation is `packaging.version.parse`.
